lib/developer_portal/lib/liquid/drops/api_spec.rb

```diff
@@ -16,6 +16,11 @@ def system_name
         @spec.system_name
       end
 
+      desc 'Returns published status true/false'
+      def published?
```

Comment:
I don't know what would be the best name for this...

The UI shows the state as "Visible" / "Hidden", but the UI property is "published".

Also not sure what is better: `published?` or just `published`.

I think `published?` is good. But what do you think @3scale/system ?

Reply:
yes I think published? is good as we see its in UI also so it won't confuse

Reply:
I think the standard in ruby is to suffix with `?` any method that returns boolean. So I vote for `published?`

Reply:
Yeah, I'd go with `published?`